docs/modules/ROOT/pages/mgmt_services/node_metadata_manager.adoc

```diff
@@ -1,19 +1,40 @@
 = Node Metadata Manager Design Specification
-Eunju Kim <eunju.kim@futurewei.com>, Liguang Xie <lxie@futurewei.com>
+Eunju Kim <eunju.kim@futurewei.com>, Liguang Xie <lxie@futurewei.com>, Prasad Kommoju <pkommoju@futurewei.com>
 v0.1, 2020-04-30
+v.02, 2021-03-15
 :toc: right
 
-NOTE: This document is under development
+NOTE: This revision includes only the new API.
 
 == Overview
 
-The node manager is a control plane component responsible of managing and configurating physical servers/nodes/hosts inside a data center.
+The node manager is a control plane component responsible for managing and configuring physical servers/nodes/hosts inside a data center. This is referred to as NMM in this and other documents.
+
 It maintains the detailed configuration for each node and collect its health state continuously in a configurable interval.
-The node health states could be used collectively in many user scenarios, for example, determine if a deployment of network resource (for example, port) is successful, or if a load balancing backend is reachable.
+The node health states could be used collectively in many user scenarios, for example, to determine if a deployment of network resource (for example, port) is successful, or if a load balancing backend is reachable.
+
+A user with administrator privileges is assumed to exist (Admin user or just Admin). The Admin interacts with Alcor controller (NodeController/NMM) through an API to register, un-register, query, and update node metadata entries.
+
+NMM API is being extended to interface with the new service called NCM. The new API allow creation, deletion, updating and querying of metadata about NCM entities. An NCM is identified by an ID and a URI. The administrator is responsible to provide a URI for the new NCM being added. There is a provision to specify the maximum number of nodes an NCM is expected to handle, also provided by the Admin.
```

Comment:
"interface with the new service called NCM" 
Recommend to change to 
=> manage the registration of a new service xref:network_config_manager.adoc[Network Configuration Manager] (NCM)

Reply:
Note that the NCM design doc is under review and has not been merged to master. Once it is merged, we can click this click and switch to NCM doc.

Reply:
I am updating the document and will push it soon.